bowling/src/frameParser.cpp

```diff
@@ -0,0 +1,54 @@
+#include "frameParser.hpp"
+
+#include <iostream>
+#include <iterator>
+#include <algorithm>
+
+parsedFrame FrameParser::parse(const std::string& gamePoints) {
+  std::vector<size_t> parsed {};
+  size_t last_parsed_char{};
+  auto tokenized = Tokenizer::splitToFramesAlt(gamePoints);
+  std::transform(tokenized.second.cbegin(),
+		 tokenized.second.cend(),
```

Comment:
bad idea to modify item of pair like this. very bad

Reply:
Could you elaborate, why? Maybe some source, materials. Im thinking about leaving pairs at all in favour of just simple vector where first element will be the name - im just little bit worried for example about strings without names.

Reply:
Becouse as reader i dont know what this pair holds,  better way is to use separate variables for name and array

Reply:
Done.